packages/gravity-ui-nunjucks/src/_patterns/04-templates/00-page-layouts/00-basic-page.njk

```diff
@@ -1,6 +1,6 @@
 {%- include 'organisms-page-header' -%}
 {%- block pageContent %}
-<main>
+<main class="grav-u-pb-xl">
```

Comment:
Hmm.. I know we agreed to add padding utils, but were they the solution we settled on for space between `main` and page footer?

I thought removing a current exception to the `* + *` rule, so that it applies to these elements was still being considered too. Or not?

Reply:
My main motivation for adding the utility classes was to fix the footer problem. It seemed an ok solution to me and is what I've seen done on other sites. Can discuss when you're back ;)

Reply:
So, I decided to remove this from this PR for now and we can fix it in another PR. I still have a requirement for these classes now aside from this issue.